### ISSUE TITLE
fatal error: snitch/snitch_registry.hpp: No such file or directory

### ISSUE THREAD

**Issue by tocic** [cschreib/snitch#90]
Trying to build an example from the README using the `1.1.0` release, I've got the following error:

```
/usr/include/snitch/snitch_macros_check.hpp:9:10: fatal error: snitch/snitch_registry.hpp: No such file or directory
    9 | #include "snitch/snitch_registry.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I believe the problem is that this file is not listed here:

https://github.com/cschreib/snitch/blob/d040b89227a4cea74986aeaf4b57ec62994cab7e/CMakeLists.txt#L68-L69

That's why it's not installed here:

https://github.com/cschreib/snitch/blob/d040b89227a4cea74986aeaf4b57ec62994cab7e/CMakeLists.txt#L99-L101

I'd provide a PR for that, but perhaps this change affects the Benchmark section too.

**Comment by cschreib** [comment]
Thank you very much for reporting this. I think you're diagnostic is correct, although I don't think this will impact the benchmark (it's done using `FetchContent`, not the CMake-installed library). I don't actually test the CMake-installed library; that needs to change so we can prevent this from happening again. I'm happy for you to do a PR to just add the missing header, so we can quickly issue v1.1.1 (that didn't take long). I can work on adding more testing in parallel, as that's lower priority.
